### ISSUE TITLE
Dialog: Missing Aria Label on Maximize button

### ISSUE THREAD

**Issue by WTW-Steve-Martin** [primefaces/primeng#14949]
### Describe the bug

The Axe Dev Tools for Accessibility is reporting the following issue:
![image](https://github.com/primefaces/primeng/assets/153538315/5ad39dc4-f3a8-4597-a277-83cd39b8f7a9)

Axe is highlighting that the Maximize button doesn't have an Aria Label

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Install the Axe Dev Tools for Accessibility plug-in.
2. Visit the PrimeNG dialog page (https://primeng.org/dialog) on Chrome/Edge browser.
3. Scan the page with Axe

### Expected behavior

The Axe scan doesn't return this error.

**Comment by WTW-Steve-Martin** [comment]
Hi,
Has this issue been assigned a milestone please?
